### PR TITLE
improved CLI installation instructions in quickstart

### LINKED ISSUE #837: docs: improve CLI installation instructions in quickstart
At present, it's pretty much "download it from the releases page."

This is _probably_ ok for most of the intended audience, but we could still afford to improve this a bit.

### PR DESCRIPTION
**Pull request by jaimin001**
Fixes #837         

Added more instructions for the CLI installation involving download of release package and adding system's `PATH` environment variable.

### CODE REVIEW

File: docs/docs/20-quickstart.mdx [hunk 1]
```diff
@@ -25,7 +25,14 @@ This guide presents a basic introduction to Kargo. Together, we will:
...
 * [Helm](https://helm.sh/docs/): These instructions were tested with v3.11.2.
-* The `kargo` CLI: Download it from [the releases page](https://github.com/akuity/kargo/releases/latest).
+* The `kargo` CLI: To install the kargo CLI, follow these steps.
+
+    1. Visit [the releases page](https://github.com/akuity/kargo/releases/latest) of the Kargo GitHub repository
+    2. Download the appropriate release package for your operating system (e.g., `kargo-linux-amd64` for Linux).
+    3. Extract the contents of the downloaded package to a directory of your choice.
+    4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.
+
+    After completing these steps, you should be able to run the kargo command in your terminal to interact with Kargo.
```
Comment (wojtekidd) [inline docs/docs/20-quickstart.mdx:35]:
```suggestion
    After completing these steps, you should be able to run the `kargo` command in your terminal to interact with Kargo.
```

**Review by wojtekidd — APPROVED** [review]

File: docs/docs/20-quickstart.mdx [hunk 2]
```diff
@@ -25,7 +25,14 @@ This guide presents a basic introduction to Kargo. Together, we will:
     * kind: v0.17.0
     * k3d: v5.4.9
 * [Helm](https://helm.sh/docs/): These instructions were tested with v3.11.2.
-* The `kargo` CLI: Download it from [the releases page](https://github.com/akuity/kargo/releases/latest).
+* The `kargo` CLI: To install the kargo CLI, follow these steps.
+
+    1. Visit [the releases page](https://github.com/akuity/kargo/releases/latest) of the Kargo GitHub repository
```
Comment (morey-tech) [inline docs/docs/20-quickstart.mdx:30]:
```suggestion
    1. Visit [the releases page](https://github.com/akuity/kargo/releases/latest) of the Kargo GitHub repository.
```

File: docs/docs/20-quickstart.mdx [hunk 3]
```diff
@@ -25,7 +25,14 @@ This guide presents a basic introduction to Kargo. Together, we will:
...
+    2. Download the appropriate release package for your operating system (e.g., `kargo-linux-amd64` for Linux).
+    3. Extract the contents of the downloaded package to a directory of your choice.
+    4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.
```
Comment (morey-tech) [inline docs/docs/20-quickstart.mdx:33]: This is great for now! However, we should include the commands to complete these steps. That would probably be better served in its own "Installing the CLI" section of the docs.

**Review by morey-tech — APPROVED** [review]

File: docs/docs/20-quickstart.mdx [hunk 4]
```diff
@@ -25,7 +25,14 @@ This guide presents a basic introduction to Kargo. Together, we will:
     * kind: v0.17.0
     * k3d: v5.4.9
 * [Helm](https://helm.sh/docs/): These instructions were tested with v3.11.2.
-* The `kargo` CLI: Download it from [the releases page](https://github.com/akuity/kargo/releases/latest).
+* The `kargo` CLI: To install the kargo CLI, follow these steps.
```
Comment (morey-tech) [inline docs/docs/20-quickstart.mdx:28]:
```suggestion
* The `kargo` CLI: To install it, follow these steps.
```

File: docs/docs/20-quickstart.mdx [hunk 5]
```diff
@@ -25,7 +25,14 @@ This guide presents a basic introduction to Kargo. Together, we will:
     * kind: v0.17.0
     * k3d: v5.4.9
 * [Helm](https://helm.sh/docs/): These instructions were tested with v3.11.2.
-* The `kargo` CLI: Download it from [the releases page](https://github.com/akuity/kargo/releases/latest).
+* The `kargo` CLI: To install it, follow these steps.
+
+    1. Visit [the releases page](https://github.com/akuity/kargo/releases/latest) of the Kargo GitHub repository.
+    2. Download the appropriate release package for your operating system (e.g., `kargo-linux-amd64` for Linux).
+    3. Extract the contents of the downloaded package to a directory of your choice.
```
Comment (krancour) [inline docs/docs/20-quickstart.mdx:32]: There's nothing to extract. The downloads are not archives.

File: docs/docs/20-quickstart.mdx [hunk 6]
```diff
@@ -25,7 +25,14 @@ This guide presents a basic introduction to Kargo. Together, we will:
...
+    4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.
```
Comment (morey-tech) [inline docs/docs/20-quickstart.mdx:33]: Addressing @krancour's comment. Nothing to extract, instead instruct to move binary to a `bin` directory
```suggestion
    3. Move the binary to a directory of your choice (e.g. for Linux, consider `/usr/local/bin/`).
    4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.
```

Comment (krancour) [reply docs/docs/20-quickstart.mdx:33]: This suggested change didn't get merged.

**Pull request merged**